packages/components/src/ui/styles/test/styled.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { css, styled } from '@wp-g2/styles';
+import { render } from '@testing-library/react';
+
+describe( 'basic', () => {
+	test( 'should create a styled component', () => {
+		const Component = styled.div`
+			background: blue;
+		`;
+
+		const { container } = render( <Component /> );
+
+		expect( container.firstChild ).toHaveStyle( { background: 'blue' } );
+	} );
+
+	test( 'should style an existing component', () => {
+		const Previous = ( props ) => <div { ...props } />;
+
+		const Component = styled( Previous )`
+			background: blue;
+		`;
```

Comment:
Should we make this test explicitly pass through `className` to make it clear what we care about here/what the qualifications are? Right now the test is ambiguous about what it cares about.

Reply:
Oh! That's a great suggestion :D